### DISCUSSION TITLE
Potion menu entry swapper

### DISCUSSION THREAD

**Discussion by popsicle7** [runelite/runelite#16453] [Q&A]
Was training herblore when i changed my left click to "Use" to make sure i don't accidentally drink the potions. Now when i went to change it back to "Drink", it was not an option for the left click, also not even shift-left click.

<img width="187" alt="Screenshot 2023-04-01 113518" src="https://user-images.githubusercontent.com/18577973/229299305-7825936f-fdae-4962-8ecb-e5437ad5ff1b.png">
<img width="195" alt="Screenshot 2023-04-01 113949" src="https://user-images.githubusercontent.com/18577973/229299309-18018167-90fd-47e7-a109-47ce2ea52b13.png">

**Comment by Felanbird** [comment]
Click `Reset`
